tests/link/pytorch/test_basic.py

```diff
@@ -301,3 +303,18 @@ def test_pytorch_MakeVector():
     x_fg = FunctionGraph([], [x])
 
     compare_pytorch_and_py(x_fg, [])
+
+
+def test_ScalarLoop():
+    n_steps = int64("n_steps")
+    x0 = float64("x0")
+    const = float64("const")
+    x = x0 + const
+
+    op = ScalarLoop(init=[x0], constant=[const], update=[x])
```

Comment:
should test one ScalarLoop with is_while as well. And I would like to actually test an Elemwise form of the ScalarLoop, since that's the most common code. After defining the `op` you pass it to an `Elemwise`, and then you can call it with tensors, instead of scalars.

Reply:
The elemwise version is kinda blocked by #1031 atm.

Reply:
I did this, give it a look